lib/RoutingList/RoutingListEdit/RoutingListEdit.js

```diff
@@ -0,0 +1,92 @@
+import { FormattedMessage } from 'react-intl';
+import { useParams } from 'react-router';
+
+import {
+  ConfirmationModal,
+  LoadingView,
+} from '@folio/stripes/components';
+
+import {
+  useShowCallout,
+  useToggle,
+} from '../../hooks';
+import {
+  useGoBack,
+  useRoutingList,
+  useRoutingListMutation,
+} from '../hooks';
+import { RoutingListForm } from '../RoutingListForm';
+import { handleRoutingListError } from '../utils';
+
+export const RoutingListEdit = () => {
+  const showCallout = useShowCallout();
+  const { id } = useParams();
+  const { routingList, isLoading } = useRoutingList(id);
+  const onClose = useGoBack(routingList?.poLineId);
+  const { deleteRoutingList, updateRoutingList } = useRoutingListMutation();
+
+  const [isDeleteConfirmationVisible, toggleDeleteConfirmation] = useToggle(false);
+
+  const onMutationSuccess = (messageId = 'stripes-acq-components.routing.list.update.success') => {
+    onClose();
+    showCallout({ messageId });
+  };
+
+  const onDelete = async () => {
+    toggleDeleteConfirmation();
+    await deleteRoutingList(
```

Comment:
the same as for https://github.com/folio-org/stripes-acq-components/pull/779/files#diff-490be3cc9d9ca478a7f11020807b408aef175f0b7411486a7d32bd046c1aeea9R24